docs/data/joy/components/circular-progress/circular-progress.md

```diff
@@ -68,6 +68,27 @@ The `determinate` prop lets you indicate a specified wait time.
 
 {{"demo": "CircularProgressDeterminate.js"}}
 
+### Determinate Count Up
```

Comment:
```suggestion

## Common examples

### Count up feature
```

I think we can create a Common examples section for this one ⎯ make sure to move it below the CSS variables playground for consistency with other pages.